internal/db/schema/migrations/postgres/2/02_empty_scope.up.sql

```diff
@@ -0,0 +1,171 @@
+begin;
+
+alter domain wt_scope_id
+  drop constraint wt_scope_id_check;
+
+alter domain wt_scope_id
```

Comment:
I'm not sure how much value that would add since it's a generated field. Plus, if we set a max length and need to increase it, the migration could take a while since the database will need to check every ID.